### DISCUSSION TITLE
how to change the parent folder?

### DISCUSSION THREAD

**Discussion by zaeboba** [tomasklaen/uosc#1014] [Q&A] [answered]
Usually nobody stores anything in the user's folder, so I'd like to change it to another drive
so you can open files directly from the player
![изображение](https://github.com/user-attachments/assets/545749df-61c0-42e1-9d84-4abd0d81d6c7)

**Comment by tomasklaen** [comment]
There's `default_directory` config for that.

**Reply by zaeboba** [reply]
Is it in mpv.conf? If so, it doesn't work for me.
![изображение](https://github.com/user-attachments/assets/e46e192a-2e68-4434-b06b-66d7575178da)
--default_directory="M:\Movie" or --default_directory=M:\Movie --default_directory=M:/Movie

**Reply by tomasklaen** [reply, marked as answer]
It's a uosc config documented in the default `uosc.conf` file.
